<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1836</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1680</width>
    <height>1050</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,0,2,0,2,0">
    <property name="rightMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="clock_layout">
      <item>
       <widget class="QLabel" name="clock_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>60</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0)</string>
        </property>
        <property name="text">
         <string>Clock</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="clock_text">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>HH:MM:SS</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="elapsed_layout">
      <item>
       <widget class="QLabel" name="elapsed_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>60</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(43, 255, 92)</string>
        </property>
        <property name="text">
         <string>Section Time Elapsed</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="elapsed_text">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(43, 255, 92);</string>
      </property>
      <property name="text">
       <string>H:MM:SS</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="remaining_layout">
      <item>
       <widget class="QLabel" name="remaining_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>60</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(226, 0, 0);</string>
        </property>
        <property name="text">
         <string>Section Time Remaining</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="remaining_sublayout" stretch="5,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>200</pointsize>
           </font>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="min_label">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>40</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(226, 0, 0)</string>
          </property>
          <property name="text">
           <string>MIN        SEC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="remaining_text">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(226, 0, 0);</string>
      </property>
      <property name="text">
       <string>H:MM:SS</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>elapsed_label</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>remaining_text</zorder>
   <zorder>elapsed_text</zorder>
   <zorder>clock_text</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
